setup.py

```diff
@@ -26,8 +26,8 @@ def text_from_file(path):
     author_email="fx-data-dev@mozilla.org",
     description="Runs a thing that analyzes experiments",
     url="https://github.com/mozilla/pensieve",
-    packages=["pensieve", "pensieve.tests", "pensieve.tests.integration"],
-    package_data={"pensieve": ["*.toml"], "pensieve.tests": ["data/*"]},
+    packages=["pensieve", "pensieve.config", "pensieve.tests", "pensieve.tests.integration"],
+    package_data={"pensieve.config": ["*.toml"], "pensieve.tests": ["data/*"]},
```

Comment:
Not sure if `config` is a good name for the directory since we also have `config.py`

Reply:
Seems fine; I think we'll want to consume these from https://github.com/mozilla/rapid-experiments-shared soon, which will give us a chance to rethink this.